ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -37,9 +37,19 @@
 
 #include <nnfw.h>
 
+/** backends supported by nnfw */
+#define NNFW_CPU_BACKEND  "cpu"
+#define NNFW_GPU_BACKEND  "acl_cl"
+#define NNFW_NEON_BACKEND "acl_neon"
+#define NNFW_SRCN_BACKEND  "srcn"
+#define NNFW_DEFAULT_BACKEND NNFW_CPU_BACKEND
```

Comment:
Q. Where are these defines used?

Reply:
A. Only the CPU backend is used currently as it is set as default. Other backends will be used in future, when the support for various backends of NNFW will be completed.